### ISSUE TITLE
vagrant package fails vmrun command

### ISSUE THREAD

**Issue by davidloone** [hashicorp/vagrant-vmware-desktop#86]
### Vagrant version

v2.3.4

### Vagrant VMware plugin version

```
vagrant-hostmanager (1.8.9, global)
  - Version Constraint: > 0
vagrant-reload (0.0.1, global)
  - Version Constraint: > 0
vagrant-vmware-desktop (3.0.2, global)
  - Version Constraint: 3.0.2
```

### Vagrant VMware utility version

1.0.21

### Host operating system

macOS Ventura 13.3.1 (a) 

VMware fusion v13.0.2

### Guest operating system

Ubuntu

### Vagrantfile

N/A

### Debug output

https://gist.github.com/davidloone/c4b4a8a98bbb754d18e90ee722b37922

Two files in gist:

- `debug_v3.0.2.txt`: Gist contains output from v3.0.2 of the VMware provider (**fails**).
- `debug_v3.0.1.txt`: Then rolled back to v3.0.1 using `vagrant plugin install --plugin-version=3.0.1 vagrant-vmware-desktop` and re-ran the same command to get output for v3.0.1 of plugin (**works**).

### Expected behavior

Vagrant box should be packaged (as per plugin v3.0.1).

### Actual behavior

```
% vagrant package ubuntu --output=ubuntu.box
==> ubuntu: Exporting VM...
An error occurred while executing `vmrun`, a utility for controlling
VMware machines. The command and output are below:

Command: ["-T", "fusion", "clone", "/Users/XXX/vagrant-utils/ubuntu/.vagrant/machines/ubuntu/vmware_desktop/896a29bf-97d8-4631-9a9a-bf225cc18c25/Other Linux 5.x kernel 64-bit Arm.vmx", "/Users/XXX/tmp/vagrant-package-20230509-49299-1540e4j/box.vmx", "full", {:notify=>[:stdout, :stderr]}]

Stdout: Error: Invalid snapshot name '{:notify=>[:stdout, :stderr]}'


Stderr:
```

### Steps to reproduce

1. run `vagrant package` using VMWare provider plugin v3.0.2.

To workaround, revert to VMWare provider plugin v3.0.1 using command

```bash
vagrant plugin install --plugin-version=3.0.1 vagrant-vmware-desktop
```

**Comment by melck** [comment]
I have same behavior when using experimental disk feature to resize or create disks.

I'm on ubuntu with vagrant version 2.3.4 and plugin 3.0.2. I have vmware workstation 17.0.2. 

When i revert plugin to 3.0.1, it works.

**Comment by chrisroberts** [comment]
Thanks for the report. The issue has been identified and a fix will be shipped shortly.

**Comment by chrisroberts** [comment]
This fix is included in version 3.0.3 of the vagrant-vmware-desktop plugin. Updating your plugin should resolve the issue.

Cheers!
